.github/utils/tutorials.sh

```diff
@@ -61,15 +62,50 @@ for script in $scripts_to_run; do
         echo "NOT using reduced GoT dataset!"
     fi
 
+    # FIXME Make the Python path editable (thanks espnet... -.-")
+    if [[ $make_python_path_editable == "EDITABLE" ]] && [[ "$script" == *"Tutorial17_"* ]]; then
+        sudo find $python_path/lib -type f -exec chmod 777 {} \;
+    fi
+
     if [[ "$script" == *".py" ]]; then
         time python $script
     else
         sudo $python_path/bin/ipython -c "%run $script"
     fi
+
+    if [ ! $? -eq 0 ]; then
+        failed=$failed" "$script
+    fi
+
+    # Clean up datasets and SQLite DBs to avoid crashing the next tutorial
     git clean -f
 
 done
 
 # causes permission errors on Post Cache
 sudo rm -rf data/
-sudo rm -rf /home/runner/work/haystack/haystack/elasticsearch-7.9.2/
\ No newline at end of file
+sudo rm -rf /home/runner/work/haystack/haystack/elasticsearch-7.9.2/
+
+
+if [[ $failed == "" ]]; then
+    echo ""
+    echo "##################################################################################"
+    echo "##                                                                              ##"
+    echo "##                   All tutorials executed successfully                        ##"
+    echo "##                                                                              ##"
+    echo "##################################################################################"
+    exit 0
+
+else
```

Comment:
Nit about UX: if we think it's important to have a visual feedback in the logs I think the two scenarios look too similar, I'd probably make the successful output less prominent, like a single line.

Reply:
Right! I assumed nobody would look at the logs for a successful run anyway :smile: But I'll change that, good point